### DISCUSSION TITLE
How to open a Dir I wanted to as Default?

### DISCUSSION THREAD

**Discussion by SinisterSup** [neovide/neovide#1462] [Q&A] [answered]
Whenever I Start the Neovide Client Just like an IDE its root directory opens at `$HOME/scoop/apps/neovide/0.9.0`. Which is not actually my working directory!
Please help me configure the proper root directory wherever my working directory is located! 
Since this is the root directory I'm unable to navigate using NERDTree though. 
I am using Neovide on Windows and all the neovim configurations are set and working properly. 
Someone, Please do help out 😄

Also "set autochdir" - does not work out. And when I toggle the NERDTree it still stays at the same PATH mentioned above.
#### Thank You

**Comment by alloc33** [comment]
you can run Neovide with shell script:
```sh
cd YOUR_LOCATION
exec /Applications/Neovide.app/Contents/MacOS/neovide --frame=none --multigrid
```

**Reply by SinisterSup** [reply]
Thank you @xbladesub  
But Instead of executing shell script every time ... Is there a way to manipulate the NerdTree to open by default in my working directory from Neovide?

**Comment by SinisterSup** [comment, marked as answer]
Well If not opening a directory by default, we can open any given folder or file with Neovide by writing a new registry file just as below!    
and Run it after saving It as a (.reg) file     
##### Relating to Windows 10/11  
Refer [this](https://dev.to/yashguptaz/add-open-with-neovide-option-in-windows-10-1cej) for further details
```
Windows Registry Editor Version 5.00
; Open files
[HKEY_CLASSES_ROOT\*\shell\Open with Neovim]
@="Edit with Neovim"
"Icon"="C:\\Program Files\\neovide\\neovide.exe,0"
[HKEY_CLASSES_ROOT\*\shell\Open with Neovim\command]
@="\"C:\\Program Files\\neovide\\neovide.exe\" \"%1\""

; This will make it appear when you right-click ON a folder
; The "Icon" line can be removed if you don't want the icon to appear
[HKEY_CLASSES_ROOT\Directory\shell\neovide]
@="Open Folder as Neovim Project"
"Icon"="\"C:\\Program Files\\neovide\\neovide.exe\",0"
[HKEY_CLASSES_ROOT\Directory\shell\neovide\command]
@="\"C:\\Program Files\\neovide\\neovide.exe\" \"%1\""

; This will make it appear when you right-click INSIDE a folder
; The "Icon" line can be removed if you don't want the icon to appear
[HKEY_CLASSES_ROOT\Directory\Background\shell\neovide]
@="Open Folder as Neovim Project"
"Icon"="\"C:\\Program Files\\neovide\\neovide.exe\",0"
[HKEY_CLASSES_ROOT\Directory\Background\shell\neovide\command]
@="\"C:\\Program Files\\neovide\\neovide.exe\" \"%V\""
```
